.gitattributes

```
@@ -0,0 +1 @@
+* text eol=lf
```

```suggestion
* -crlf
```
is closer to what we want as it just disables any automatic conversion without forcing a particular type of line ending.